### ISSUE TITLE
Prerelease web tool requires at least one released repo

### ISSUE THREAD

**Issue by mxgrey** [ros-infrastructure/prerelease_website#48]
If you want to test a prerelease of a repo that has not already been added to the ros distro, you need to include at least existing repo to the list, or else you cannot proceed past step 2.

For people trying to test a prerelease of one or more new repos before adding them to a distro, it would be useful if the tool could work with just a list of custom repositories.

To recreate this issue, simply run the web tool and only add a custom repository. The tool will quietly freeze when you attempt to click `Next` at the second step.

**Comment by wjwwood** [comment]
I believe you can add a custom repository and remove the default first entry and then you don't need anything to be released.

**Comment by mxgrey** [comment]
It should probably hang at the first `Next` and highlight that first entry with a suggestion to either fill it in or delete it before proceeding.

**Comment by wjwwood** [comment]
Yeah, that would be better, currently it just has a JS error on the next button of the second page. We could also just make the JS code robust to empty entries.

**Comment by sloretz** [comment]
ROS 1 is end-of-life (EOL) as of today, May 31st 2025. I am archiving this repository because:

* it only supports ROS 1
* it isn't needed anymore in ROS 2

[If you still rely on ROS 1, read this page to learn about your options.](https://bit.ly/NoeticEOL)
